packages/http-client-csharp/generator/Microsoft.TypeSpec.Generator/test/TestHelpers/MockHelpers.cs

```diff
@@ -69,6 +70,8 @@ public static Mock<CodeModelPlugin> LoadMockPlugin(
             // initialize the singleton instance of the plugin
             var mockPlugin = new Mock<CodeModelPlugin>(new GeneratorContext(Configuration.Load(configFilePath, configuration))) { CallBase = true };
 
+            mockPlugin.Setup(p => p.Emitter).Returns(new Emitter(Console.OpenStandardOutput()));
```

Comment:
I did not add a hook here so that we could mock it in our UT - but if in the future we need to check something, we could do it here.